src/benchmarklib/benchmark_runner.cpp

```diff
@@ -138,6 +138,13 @@ void BenchmarkRunner::_benchmark_individual_queries() {
     result.iteration_durations = state.iteration_durations;
 
     _query_results_by_query_name.emplace(name, result);
+
+    const auto duration_ns = std::chrono::duration_cast<std::chrono::nanoseconds>(result.duration).count();
```

Comment:
Use format_duration

Reply:
That is the same format as in the JSON - I believe that makes it easier to compare.

Reply:
oookay

Reply:
approve plz?